assets/package.json

```diff
@@ -1,20 +1,33 @@
 {
   "scripts": {
-    "deploy": "brunch build --production",
-    "watch": "brunch watch --stdin"
+    "build": "cross-env NODE_ENV=production webpack",
```

Comment:
The name chosen is deploy [such as to be consistent with the Phoenix deployment guidelines](https://hexdocs.pm/phoenix/deployment.html#compiling-your-application-assets)